snakemake/remote/GS.py

```diff
@@ -197,7 +197,7 @@ async def inventory(self, cache: snakemake.io.IOCache):
     # === Implementations of abstract class members ===
 
     def get_inventory_parent(self):
-        return self.bucket_name
+        return "%s/%s" % (self.bucket_name, os.path.dirname(self.blob.name))
```

Comment:
Let us rather use an f-string here:
```suggestion
        return f"{self.bucket_name}/{os.path.dirname(self.blob.name)}"
```